ph5/clients/tests/test_ph5tostationxml.py

```diff
@@ -223,8 +223,9 @@ def test_create_obs_network(self):
         with LogCapture() as log:
             log.setLevel(logging.WARNING)
             self.parser.create_obs_network()
+            errmsgs = sorted(self.errmsgs)
             for i in range(len(self.errmsgs)):
-                self.assertEqual(log.records[i].msg, self.errmsgs[i])
+                self.assertEqual(log.records[i].msg, errmsgs[i])
```

Comment:
```suggestion
      self.assertEqual(set(rec.msg for rec in log.records), set(self.errmsgs))
```

Reply:
Thank you for your suggestion. I have never tried to use set before.